keras/ops/core_test.py

```diff
@@ -814,3 +814,212 @@ def test_cond_check_output_spec_tuple(self):
                 (mock_spec,), (mock_spec, mock_spec_different)
             )
         )
+
+    def test_unstack_with_negative_axis_first_tensor(self):
+        x = np.random.rand(4, 3, 2)
+        x_tensor = ops.convert_to_tensor(x)
+        negative_axis = -1
+        result = ops.unstack(x_tensor, axis=negative_axis)
+        expected_shape = (4, 3)
+        self.assertAllEqual(result[0].shape, expected_shape)
+
+    def test_unstack_with_negative_axis_second_tensor(self):
+        x = np.random.rand(4, 3, 2)
+        x_tensor = ops.convert_to_tensor(x)
+        negative_axis = -1
+        result = ops.unstack(x_tensor, axis=negative_axis)
+        expected_shape = (4, 3)
+        self.assertAllEqual(result[1].shape, expected_shape)
+
+    def test_cond_output_spec_mismatch_true_fn_dict_false_fn_not_dict(self):
+        cond_op = core.Cond()
+
+        # true_fn returns a dictionary
+        def true_fn():
+            return {"a": KerasTensor(shape=(3, 3), dtype="float32")}
```

Comment:
It is not correct usage to instantiate and return a `KerasTensor` here (and the other examples below). Prefer using e.g. `ops.ones()`.